scripts/fast-reboot

```diff
@@ -174,6 +175,10 @@ function initialize_pre_shutdown()
 
 function request_pre_shutdown()
 {
+    if [ -x ${DEVPATH}/${PLATFORM}/${PLATFORM_REBOOT_PRE_CHECK} ]; then
+        debug "Requesting platform reboot pre-check ..."
+    	${DEVPATH}/${PLATFORM}/${PLATFORM_REBOOT_PRE_CHECK} ${REBOOT_TYPE} 
```

Comment:
What is this pre-check script?

What should be desired action when this script returns non-zero value or fails?

If the intention is to abort the reboot procedure:
1. Such error should be logged by checking return code, and calling `debug` message.
2. This check may be moved to `reboot_pre_check` as that's where we decide if the device is healthy enough for a reboot.

Reply:
This is a script that does platform specific cleanup. In our case, it currently terminates a platform specific process in syncd conatiner. It is best effort, so we dont care the return code of the script.